package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "ecommerce-app",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "nodemon ./bin/www",
+    "db:makemigrations": "./node_modules/sequelize-auto-migrations/bin/makemigration.js"
+  },
+  "dependencies": {
+    "basic-auth": "^2.0.1",
```

Comment:
@osama891 Why this package?

Reply:
i thought about extracting login info from header using basic-auth but the encrypter that i was using was not suitable for this. I'll remove this package